react-components/src/components/CacheProvider/FdmNodeCache.ts

```diff
@@ -317,70 +293,70 @@ export class FdmNodeCache {
 }
 
 async function createRevisionToEdgesMap(
```

Comment:
Maybe it is better to change any `RevisionToEdgesMap` wording to something like `RevisionToConnectionsMap` as well